test/resources/controls/aws_parallelcluster_install/mysql_client_spec.rb

```diff
@@ -16,20 +16,22 @@
   if os.redhat?
```

Comment:
Nitpick: if we are using `os_properties.ubuntu?` below, maybe we should use `os_properties.redhat?` here too?

Reply:
No, that would be wrong: `os.redhat?` actually checks if the os is a RHEL derivative, and is evaluated to true even for CentOS and Amazon Linux 2. So better to leave it as it is.

Reply:
`os` is a native resource and it doesn't have `ubuntu?` [helper](https://docs.chef.io/inspec/resources/os/#osfamily-helpers), thus the difference. See the comment above wrt the choice between `ubuntu?` and `debian?`.